service/src/main/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsService.java

```diff
@@ -249,7 +249,7 @@ public HearingBooking getHearingBooking(UUID id, List<Element<HearingBooking>> h
 
     public void findAndSetPreviousVenueId(CaseData caseData) {
         if (isNotEmpty(caseData.getHearingDetails()) && caseData.getPreviousHearingVenue() != null
-            && caseData.getPreviousHearingVenue().getUsePreviousVenue().equals(YES.getValue())) {
+            && YES.getValue().equals(caseData.getPreviousHearingVenue().getUsePreviousVenue())) {
```

Comment:
Could `caseData.getPreviousHearingVenue()` have a chance to be null?

Reply:
well if it is then the condition will exit before this part, we check that it's not null

Reply:
Completely missed that, too early 😅